worker/dbaccessor/worker_integration_test.go

```diff
@@ -69,6 +69,7 @@ var _ = gc.Suite(&integrationSuite{})
 
 func (s *integrationSuite) SetUpTest(c *gc.C) {
 	s.DqliteSuite.SetUpTest(c)
+	s.dqliteAppIntegrationSuite.SetUpTest(c)
```

Comment:
`DqliteSuite.SetUpTest` should be called in `dqliteAppIntegrationSuite.SetUpTest` as it is embedded in that struct. Composing `SetUpTest` from multiple embedded layers is a smell.

Reply:
I'd agree, but you're actively not calling the dqliteAppIntegrationSuite parent suite. So you need to call setup test to get the skip to behave correctly. It's smell, but it's only used in this one location.